setup.py

```diff
@@ -112,7 +112,7 @@ def run_tests(self):
         "all": ["requests[security]"]
     },
     tests_require=[
-        "pytest", "mock"
+        "pytest", "mock", "pytest-mock", "pytest-localserver"
```

Comment:
See PR https://github.com/spotify/ramlfications/pull/144; I'll rebase this if you land that separately.